### DISCUSSION TITLE
Nuxt-config option "extends" questions

### DISCUSSION THREAD

**Discussion by raven-dever** [nuxt/nuxt#16154] [Questions]
Is it possible to include/resolve dependencies from the external project without node_modules-folder?
![image](https://user-images.githubusercontent.com/93713466/200873027-80b3e5d6-6007-411c-8b70-13341172d03d.png)

And how can you extend from a remote git-Service like Github? The current Docs aren't very clear on this:
https://v3.nuxtjs.org/api/configuration/nuxt-config#extends
I have a simple base app on Github public: https://github.com/GregorKoglbauer/nuxt-3-base-app
I tried the following strings, but none of them work:
'https://github.com/GregorKoglbauer/nuxt-3-base-app.git'
'github:GregorKoglbauer/nuxt-3-base-app.git'
'github:https://github.com/GregorKoglbauer/nuxt-3-base-app.git'

**Comment by iamsurelydaveydave** [comment]
I tried extending my base up with another remote nuxt app (private repo) and I get a 404 not found.

**Reply by gwen-bo** [reply]
Maybe a redundant reply: but have you added a Github access-token (`GIGET_AUTH`) in an `.env` file? Worked for me where I was extending my project with a remote (private) github repo. ) - as the [docs](https://nuxt.com/docs/guide/going-further/layers#git-repository) read: 
> If you want to extend a private remote source, you need to add the environment variable GIGET_AUTH=<token> to provide a token.
